mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/General.xml

```diff
@@ -23,6 +23,12 @@
             <default>all</default>
             <Required>Y</Required>
         </inet_interfaces>
+        <bind_address type="NetworkField">
```

Comment:
can we not pin this to an address family? if not the validation is leaky and may break postfix config

Reply:
You mean in temlating? if contains ':' bind6 else bind4 and only one bind field?

Reply:
maybe this is more of a question for @AdSchellevis for the NetworkField type validation :)

Reply:
that's exactly why I mention it... the validation is not correct and it will create support efforts for other people who possibly cannot read but use advanced settings ... ;)

Reply:
@fichtner How to go on with this? @AdSchellevis Is validation with mask only for TextField? I tried this for NetworkField:

v4
<mask>/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/</mask>
v6
<mask>/^([0-9a-fA-F\:]){1,40}$/u</mask>

But doesn't seem to work ...